iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -33,52 +33,102 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
 {
     senderRuntime->offerService({"service1", "instance1"});
     this->InterOpWait();
+
+    auto instanceContainer = receiverRuntime->findService({"service1", "instance1"});
+
+    ASSERT_THAT(instanceContainer.get_value().size(), Eq(1u));
+    ASSERT_THAT(*instanceContainer.get_value().begin(), Eq(IdString("instance1")));
+}
+
+
+///@todo #359 offereService should return bool signalling success/failure
```

Comment:
Can you maybe integrate this follow-up in this PR here directly? It should be not much work to change the signature.

Reply:
okay